src/lib/components/Descope.tsx

```diff
@@ -36,13 +37,26 @@ const DescopeWC = lazy(async () => {
 				theme={theme}
 				debug={debug}
 				telemetryKey={telemetryKey}
+				redirect-url={redirectUrl}
```

Comment:
Let's add it to the readme configuration table 

Also I wonder it we should add an env variable to the example app, but less critical

Reply:
Added for both.

Reply:
Tested and it works good.